x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/http/sender/RequestTaskTests.java

```diff
@@ -57,144 +39,74 @@
 public class RequestTaskTests extends ESTestCase {
 
     private static final TimeValue TIMEOUT = new TimeValue(30, TimeUnit.SECONDS);
-    private final MockWebServer webServer = new MockWebServer();
     private ThreadPool threadPool;
 
     @Before
     public void init() throws Exception {
-        webServer.start();
         threadPool = createThreadPool(inferenceUtilityPool());
     }
 
     @After
     public void shutdown() {
         terminate(threadPool);
-        webServer.close();
     }
 
-    public void testDoRun_SendsRequestAndReceivesResponse() throws Exception {
```

Comment:
These aren't applicable anymore since the `RequestTask` isn't executable anymore